app/controllers/api/v1/workflows_controller.rb

```diff
@@ -15,10 +12,18 @@ def show
 
   private
 
+  def context
+    case action_name
+    when "show", "index"
+      { languages: current_languages }
+    else
+      {}
+    end
+  end
+
   def create_response(workflow)
     serializer.resource({},
-                        resource_scope(workflow),
-                        languages: [workflow.primary_language])
+                        resource_scope(workflow))
```

Comment:
Actually I think these methods can be entirely removed since they no longer have class specific behaviour.